### DISCUSSION TITLE
Galera, o que vocês estão usando de framework para fazer micro-frontends? O que vocês indicam? E quando usam?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#2174] [Discussão]
Estava me aprofundando um pouco, entendendo, vi que temos o module federation do webpack, também o single spa e alguns outros framewoaks, qual vocês estão vendo como sendo o mais usado? sabem falar as vantagens e desvantagens de cada?

E sobre micro-frontends mesmo, quando vocês gostam de usar e quando evitam?

Gostaria de opiniões, muito obrigado galera!
Forte abraço!!

**Comment by igorlvicente** [comment]
A minha conclusão do pouco que vi sobre isso é que a complexidade e as desvantagens não valem a pena pra nenhum dos projetos em que eu já trabalhei.
Posso estar errado. Vou acompanhar a thread.

**Comment by ninetails** [comment]
#661 , #1083

tenta perguntar pra galera que interagiu nestas threads

**Comment by wilsonneto-swe** [comment]
Muito obrigado @igorlvicente , e já lendo as threads aqui @ninetails , muito obrigado!! 
Abraços!!
